lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -388,11 +388,11 @@ SPIRVEntry::setLinkageType(SPIRVLinkageTypeKind LT) {
   addDecorate(new SPIRVDecorateLinkageAttr(this, Name, LT));
 }
 
-std::ostream &
-operator<<(std::ostream &O, const SPIRVEntry &E) {
+llvm::raw_ostream &
+operator<<(llvm::raw_ostream &O, const SPIRVEntry &E) {
   E.validate();
   E.encodeAll(O);
-  O << SPIRVNL;
+  SPIRVNL(O);

```

Comment:
Do you want me to create a local variable of SPIRVNL type each time we need a new line in SPIRV text or create one global variable and re-use it?